default-config.toml

```diff
@@ -44,6 +44,12 @@ color = true
 # Whether ANSI color codes from the page content should be rendered
 ansi = true
 
+# Whether or not to support source code highlighting in preformatted blocks based on alt text
+highlight_code = true
+
+# Which highlighting style to use (see https://xyproto.github.io/splash/docs/)
+highlight_style = "swapoff"
```

Comment:
I find this theme pretty ugly and a poor use of color. Could it be switched to "vim"? Make sure to change the default everywhere.

Reply:
No problem, swapoff looked good with the old default theme but vim looks better now!